editor/animation_track_editor.cpp

```diff
@@ -7007,25 +7016,26 @@ void AnimationTrackEditor::_selection_changed() {
 	}
 }
 
+void AnimationTrackEditor::_update_snap_unit() {
+	if (timeline->is_using_fps() && step->get_value() > 0) {
+		snap_unit = 1.0 / step->get_value();
+	} else {
+		snap_unit = 1.0 / Math::round(1.0 / step->get_value()); // Follow the snap behavior of the timeline editor.
```

Comment:
Is there a chance that step->get_value() > 0 is false? I always like to make sure there is no scenario where we are dividing by 0.

Reply:
Oh sure. This was not enough to avoid zero div, thanks. Now fixed.